keps/sig-node/20190129-hugepages.md

```diff
@@ -279,8 +290,94 @@ opencontainers/runtime-spec.
 see:
 https://github.com/opencontainers/runtime-spec/blob/master/config-linux.md#huge-page-limits
 
+The runtime-spec says `hugepageLimits` as an array of objects, and each object
+allows specifying `limit` per `pageSize`.
+```
+    "hugepageLimits": [
+        {
+            "pageSize": "2MB",
+            "limit": 209715200
+        },
+        {
+            "pageSize": "64KB",
+            "limit": 1000000
+        }
+   ]
+```
+
+The `LinuxContainerResources` message can be extended to specify multiple size
```

Comment:
per the 9/24 sig-node meeting, there was no disagreement on enabling support for container level isolation.  as we work on the implementation of the CRI, we can adjust if needed, but in general, we know we will need to update the CRI to pass hugepage sizes down.

Reply:
`but in general, we know we will need to update the CRI to pass hugepage sizes down.`
=> Ok, I will consider it and add unittests to cover this case(hugepage sizes down) in implementation phase.

Reply:
@derekwaynecarr 
`but in general, we know we will need to update the CRI to pass hugepage sizes down`
=> Can you explain more about the case of updating the resource of a container after creating a container?
It feels like meaning `in-place update of pod resources`, is that it?
https://github.com/kubernetes/enhancements/pull/686